OSDOCS-8402: Migrate "FAQ" from rosaworkshop.io (#70661)
openshift/openshift-docs: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-what-is-rosa.adoc

* *Flexible, consumption-based pricing:* Scale to your business needs and pay as you go with flexible pricing and an on-demand hourly or annual billing model.
* *Single bill for Red Hat OpenShift and AWS usage:* Customers will receive a single bill from AWS for both Red Hat OpenShift and AWS consumption.
* *Fully integrated support experience:* Installation, management, maintenance, and upgrades are performed by Red Hat site reliability engineers (SREs) with joint Red Hat and Amazon support and a 99.95% service-level agreement (SLA).
* *AWS service integration:* AWS has a robust portfolio of cloud services, such as compute, storage, networking, database, analytics, and machine learning. All of these services are directly accessible through ROSA. This makes it easier to build, operate, and scale globally and on-demand through a familiar management interface.
* *Maximum Availability:* Deploy clusters across multiple availability zones in supported regions to maximize availability and maintain high availability for your most demanding mission-critical applications and data.
* *Cluster node scaling:* Easily add or remove compute nodes to match resource demand.

> * *Optimized clusters:* Choose from memory-optimized, compute-optimized, or general purpose EC2 instance types with clusters sized to meet your needs. For more information, see xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-aws-compute-types_rosa-service-definition[AWS compute types].

Reviewer: maybe this link to should go to instance types: https://70661--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-service-definition#rosa-sdpolicy-instance-types_rosa-service-definition ?

* *Global availability:* Refer to the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-regions-az_rosa-service-definition[product regional availability page] to see where ROSA is available globally.

== ROSA and Kubernetes

> In ROSA, everything you need to deploy and manage containers is bundled including container management, Operators, networking, load balancing, service mesh, CI/CD, firewall, monitoring, registry, authentication, and authorization capabilities. These components are tested together for unified operations as a complete platform. Automated cluster operations, including over-the-air platform upgrades, further enhance your Kubernetes experience.

Reviewer: ```suggestion
In ROSA, everything you need to deploy and manage containers is bundled, including container management, Operators, networking, load balancing, service mesh, CI/CD, firewall, monitoring, registry, authentication, and authorization capabilities. These components are tested together for unified operations as a complete platform. Automated cluster operations, including over-the-air platform upgrades, further enhance your Kubernetes experience.
```

== Basic responsibilities

> In general, cluster deployment and upkeep is Red Hat's or AWS's responsibility, while applications, users, and data is the customer's responsibility. For a more detailed breakdown of responsibilities, see the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix.adoc#rosa-policy-responsibility-matrix[responsibility matrix].

Reviewer: ```suggestion
In general, cluster deployment and upkeep is Red Hat's and/or AWS's responsibility, while applications, users, and data is the customer's responsibility. For a more detailed breakdown of responsibilities, see the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix.adoc#rosa-policy-responsibility-matrix[responsibility matrix].
```

Reviewer: Not sure if that is the best approach, but given the matrix it can be a combo of the two rather  than either/or?

Author: I am going to leave it "as is" for now because we are not supposed to use "and/or" in our docs.

Reviewer: > I am going to leave it "as is" for now because we are not supposed to use "and/or" in our docs.

Eh, good point...after I made that suggestion I realized that term is not allowed...and then forgot to change it.. :(

== Roadmap and feature requests
Visit the link:../https://red.ht/rosa-roadmap[ROSA roadmap] to stay up-to-date with the status of features currently in development. Open a new issue if you have any suggestions for the product team.

== AWS region availability
Refer to the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-regions-az_rosa-service-definition[product regional availability] page for an up-to-date view of where ROSA is available.

== Compliance certifications
ROSA is currently compliant with SOC-2 type 2, SOC 3, ISO-27001, ISO 27017, ISO 27018, HIPAA, GDPR, and PCI-DSS. We are also currently working towards FedRAMP High. 

== Nodes
=== Worker nodes across multiple AWS regions
All nodes in a ROSA cluster must be located in the same AWS region. For clusters configured for multiple availability zones, control plane nodes and worker nodes will be distributed across the availability zones.

> === Minimum number of worker nodes

Reviewer: +```suggestion
=== Minimum number of worker nodes
```

For a ROSA cluster, the minimum is 2 worker nodes for single availability zone and 3 worker nodes for multiple availability zones.
=== Underlying node operating system
As with all OpenShift v4.x offerings, the control plane, infra and worker nodes run Red Hat Enterprise Linux CoreOS (RHCOS).
=== Node hibernation or shut-down
At this time, ROSA does not have a hibernation or shut-down feature for nodes. The shutdown and hibernation feature is an OpenShift platform feature that is not yet mature enough for widespread cloud services use.
=== Supported instances for worker nodes

> For a complete list of supported instances for worker nodes see xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-aws-instance-types_rosa-service-definition[AWS compute types]. Spot instances are also supported.

Reviewer: ```suggestion
For a complete list of supported instances for worker nodes see xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-aws-instance-types_rosa-service-definition[AWS instance types]. Spot instances are also supported.
```

=== Node autoscaling
Autoscaling allows you to automatically adjust the size of the cluster based on the current workload. See xref:../../rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.adoc#rosa-nodes-about-autoscaling-nodes[About autoscaling nodes on a cluster] for more details.
=== Maximum number of worker nodes
The maximum number of worker nodes is 180 worker nodes for each ROSA cluster. See xref:../../rosa_planning/rosa-limits-scalability.adoc#rosa-limits-scalability[limits and scalability] for more details on node counts.

> 

Reviewer: Several formatting issues above. Think if you add '+' before headers that should fix the issue. 
![image](https://github.com/openshift/openshift-docs/assets/122639474/8bd4f920-ec07-4685-b030-fe8e30a1d428)

A list of the account-wide and per-cluster roles is provided in the xref:../../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-account-wide-roles-and-policies-creation-methods_rosa-sts-about-iam-resources[ROSA documentation].

== Administrators
A ROSA customer's administrator can manage users and quotas in addition to accessing all user-created projects.

== OpenShift versions and upgrades
ROSA is a managed service which is based on OpenShift Container Platform. You can view the current version and life cycle dates in the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle-dates_rosa-life-cycle[ROSA documentation].

Customers can upgrade to the newest version of OpenShift and use the features from that version of OpenShift. For more information, see xref:../../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle-dates_rosa-life-cycle[life cycle dates]. Not all OpenShift features are be available on ROSA. Review the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-service-definition[Service Definition] for more information.

== Support

> ROSA is supported by Red Hat and AWS, and you can contact the support team from either company to begin troubleshooting. Any escalations will be facilitated by Red Hat and AWS to engage the best team to address the issue.

Reviewer: Should this specified to reach out to a specific company based on the issue and then point to responsibility matrix? I would think statement as is may be an issue legally, but not sure.

* link:https://access.redhat.com/[Red Hat Support]
* link:https://aws.amazon.com/premiumsupport/[AWS Support]
** AWS support customers must have a valid AWS support contract

> 

Reviewer: Bulleted item format looks a little strange to me, but I could be wrong.

You can also visit the link:http://access.redhat.com/[Red Hat Customer Portal] to search or browse through the Red Hat knowledge base of articles and solutions relating to Red Hat products or submit a support case to Red Hat Support.

> You can also open a ticket directly from the link:https://console.redhat.com/openshift[Red Hat OpenShift Cluster Manager]. See the xref:../../support/getting-support.adoc#getting-support[ROSA support documentation] for more details about obtaining support.

Reviewer: ```suggestion
Additionally, you can open a ticket directly from the link:https://console.redhat.com/openshift[Red Hat OpenShift Cluster Manager]. See the xref:../../support/getting-support.adoc#getting-support[ROSA support documentation] for more details about obtaining support.
```

Reviewer: Not sure if "Additionally" is the best choice of words, just don't like the "You can also" 2x in a row... :)

> If a ROSA cluster is not upgraded before the "end of life" date, the cluster continues to operate in a limited support status. The SLA for that cluster will no longer be applicable, but you can still get support for that cluster. See the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[limited support status]documentation for more details.

Reviewer: ```suggestion
If a ROSA cluster is not upgraded before the "end of life" date, the cluster continues to operate in a limited support status. The SLA for that cluster will no longer be applicable, but you can still get support for that cluster. See the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[limited support status] documentation for more details.
```

== Service-level agreement (SLA)
Refer to the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-sla_rosa-service-definition[ROSA SLA] page for details.

== Notifications and communication

> Red Hat will provide notifications regarding new features, updates, and scheduled maintenance through email, including AWS updates and email,  and the Red Hat console service log.

Reviewer: ```suggestion
Red Hat will provide notifications regarding new Red Hat and AWS features, updates, and scheduled maintenance through email  and the Red Hat console service log.
```

Reviewer: Not sure if this is now accurate (although I think it is) but current phrasing is confusing IMO.

== Open Service Broker for AWS (OBSA)  
You can use OSBA with ROSA.  See link:https://aws.amazon.com/partners/servicebroker/[Open Service Broker for AWS] for more information.

> The preferred method is the more recent link:https://github.com/aws-controllers-k8s/community[AWS Controller for Kubernetes].

Reviewer: The preferred method for using OSBA?

== Offboarding  
Customers can stop using ROSA at any time and move their applications to on-premise, a private cloud, or other cloud providers. Standard reserved instances (RI) policy applies for unused RI.   

== Authentication
ROSA supports the following authentication mechanisms: OpenID Connect (a profile of OAuth2), Google OAuth, GitHub OAuth, GitLab, and LDAP.

== SRE cluster access

> All SRE cluster access is secured by MFA. See xref:../../rosa_architecture/rosa_policy_service_definition/rosa-policy-process-security.adoc#rosa-policy-sre-access_rosa-policy-process-security[SRE access] for more details.

Reviewer: Link probably needs to now go here: https://70661--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-sre-access

== Encryption

=== Encryption keys

> We encrypt EBS volumes that we use for ROSA, using a key stored in KMS. Customers also have the option to provide their own KMS keys at cluster creation.

Reviewer: ```suggestion
ROSA uses a key stored in KMS to encrypt EBS volumes. Customers also have the option to provide their own KMS keys at cluster creation.

=== Multi-region KMS keys for EBS encryption
Currently, the ROSA CLI does not accept multi-region KMS keys for EBS encryption. This feature is in our backlog for product updates. The ROSA CLI accepts single region KMS keys for EBS encryption if it is defined at cluster creation.

== Infrastructure
ROSA uses several different cloud services such as virtual machines, storage, and load balancers. You can see a defined list in the xref:../../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-aws-policy-provisioned_rosa-sts-aws-prereqs[AWS prerequisites].

== Credential methods
There are two credential methods to grant Red Hat the permissions needed to perform the required actions in your AWS account: AWS with STS or an IAM user with admin permissions. AWS with STS is the preferred method, and the IAM user method will eventually be deprecated. AWS with STS better aligns with the principles of least privilege and secure practices in cloud service resource management.
//See the section [ROSA with STS Explained] section for a detailed explanation.

== Prerequisite permission or failure errors

> Check for a newer version of the ROSA CLI. Every release of the ROSA CLI lands in two places: link:https://github.com/openshift/rosa/releases[Github] and the link:https://www.openshift.com/products/rosa/download[Red Hat signed binary releases].

Reviewer: ```suggestion
Check for a newer version of the ROSA CLI. Every release of the ROSA CLI is located in two places: link:https://github.com/openshift/rosa/releases[Github] and the link:https://www.openshift.com/products/rosa/download[Red Hat signed binary releases].
```

== Storage
Refer to the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-storage_rosa-service-definition[storage] section of the service definition.

OpenShift includes the CSI driver for AWS EFS. For more information, see xref:../../storage/container_storage_interface/osd-persistent-storage-aws-efs-csi.adoc#osd-persistent-storage-aws-efs-csi[Setting up AWS EFS for Red Hat OpenShift Service on AWS].

== Using a VPC

> You can deploy into an existing VPC and choose the specific subnets. At installation you can select to deploy to an existing VPC. You can then select the required subnets and provide a valid CIDR range that encompasses the subnets for the installation program when using those subnets.

Reviewer: First two lines seem redundant? Maybe get rid of first line? Also, can't user's deploy into an existing VPC or BYO VPC?
